query/stdlib/influxdata/influxdb/rules.go

```diff
@@ -684,7 +684,7 @@ func canPushWindowedAggregate(ctx context.Context, fnNode plan.Node) bool {
 	}
 
 	// Check the aggregate function spec. Require operation on _value. There
-	// are two feature flags covering all cases. One specifically for Count,
+	// are two feature flags covering all cases. One specifically for Count, one for Sum,
```

Comment:
Can you change this comment to something more generic like:

```
// Check the aggregate function spec. Require the operation on _value
// and check the feature flag associated with the aggregate function.
```

This way we don't have to update the comment each time we make a change to the feature flags.